### ISSUE TITLE
no-tabs: allow tabs inside strings

### ISSUE THREAD

**Issue by ficristo** [eslint/eslint#11172]
**What rule do you want to change?**
no-tabs

**Does this change cause the rule to produce more or fewer warnings?**
Fewer warnings

**How will the change be implemented? (New option, new default behavior, etc.)?**
New option

**Please provide some example code that this change will affect:**

```js
// Many spaces are tabs.
"function funcWithTabs() {\n" +
"	var i		= 0;\n" +
"	var	offset	= 0;\n" +
"}"
```

**What does the rule currently do for this code?**
Show errors.

**What will the rule do after it's changed?**
Do not show the errors.

**Are you willing to submit a pull request to implement this change?**
Unlikely
